packages/core/src/builder.class.ts

```diff
@@ -769,6 +769,11 @@ export interface Component {
    */
   models?: readonly string[];
 
+  /**
+   * Whether the component is a React Server Component
+   */
+  isRSC?: boolean;
```

Comment:
On second thought i would prefer we don't have this type here. Some customers might see this and set it to `true` in their React Gen1 SDK implementation, and then they'll be confused as to why it doesn't do anything